portfolio/src/main/webapp/style.css

```diff
@@ -64,3 +64,11 @@ ol {
 .fa-github {
   color: white;
 }
+
+#map {
+  border: thin solid black;
+  height: 500px;
+  width: 500px;
+  margin-left: auto;
```

Comment:
You can just use `margin: auto` instead of these separate lines

Reply:
Thanks for the tip will fix that!